exporter/datadogexporter/examples/kafka.yaml

```diff
@@ -0,0 +1,61 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: "localhost:4318"
```

Comment:
This should be `0.0.0.0:4318`. Same for below.

Reply:
This is intentional, setting `0.0.0.0` logs a warning. See: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks

Reply:
My idea there is to start with 0.0.0.0 (which is the implicit current state within opentelemetry.io examples) and then further restrict the endpoint in examples where we can